flank-scripts/src/main/kotlin/flank/scripts/cli/firebase/SaveServiceAccountCommand.kt

```diff
@@ -0,0 +1,16 @@
+package flank.scripts.cli.firebase
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import flank.scripts.ops.firebase.saveServiceAccount
+
+object SaveServiceAccountCommand : CliktCommand(
+    name = "save_service_account",
+    help = "Save given service account to flank credentials location"
+) {
+    private val serviceAccount: String by option("--account").required()
```

Comment:
Do we want to validate it somehow? (for example non-empty string)

Reply:
And maybe nice error handling? (not request, just thinking)

Reply:
If you did not pass the parameter it should fail on `flank-scripts` side. Other validation will be performed on Flank side

Reply:
Got it 👍 
There are two cases I meant:
* `firebase save_service_account --account=` will create an empty JSON file (the question is if we should bother 🤷 )
* `firebase save_service_account --account=asdasdsdsfas.json` throws an error with nasty stacktarce (same as above)

Reply:
Personally, I think that we should not validate the service account provided by user